topics/timeseries.md

```diff
@@ -177,11 +177,11 @@ A contributor is a user other than the owner of the content being viewed, and ty
 <br/>
 <a name="player-annotate-createannotation"></a>
 
-### Events with class `player.annotate.createannotation`, `player.annotate.createannotation.classcomment`, `player.annotate.createannotation.personalnote`, `player.annotate.createannotation.anonymouscomment`
+### Events with class `player.annotate.createannotation`, `player.annotate.createannotation.highlight`, `player.annotate.createannotation.pin`, `player.annotate.createannotation.classcomment`, `player.annotate.createannotation.personalnote`, `player.annotate.createannotation.anonymouscomment`
 
 A user has added a comment to a Talis Elevate player resource.
 
-`player.annotate.createannotation` includes a count of events for all of the sub classes `player.annotate.createannotation.classcomment`, `player.annotate.createannotation.personalnote`, `player.annotate.createannotation.anonymouscomment`. Counting all four classes will result in counting events twice.
+`player.annotate.createannotation` includes a count of events for all of the sub classes `player.annotate.createannotation.highlight`, `player.annotate.createannotation.pin`,`player.annotate.createannotation.classcomment`, `player.annotate.createannotation.personalnote`, `player.annotate.createannotation.anonymouscomment`. Counting all six classes will result in counting events twice.
```

Comment:
I think these pin drop and highlight events get emitted at the same moment as the classcomment events?

So do we need to separate these out into a separate group of events and make it clear that when aggregating event counts, they should be distinct sets?

Reply:
Correct. I've updated this description - let me know what you think.